cluster/gke/util.sh

```diff
@@ -91,21 +91,39 @@ function kube-up() {
   # Make the specified network if we need to.
   if ! gcloud compute networks describe "${NETWORK}" &>/dev/null; then
     echo "Creating new network: ${NETWORK}" >&2
-    gcloud compute networks create "${NETWORK}" --range "${NETWORK_RANGE}"
+    echo gcloud compute networks create "${NETWORK}" --project="${PROJECT}" --range "${NETWORK_RANGE}"
+    gcloud compute networks create "${NETWORK}" --project="${PROJECT}" --range "${NETWORK_RANGE}"
+  else
+    echo "Using network: ${NETWORK}"
   fi
 
   # Allow SSH on all nodes in the network. This doesn't actually check whether
   # such a rule exists, only whether we've created this exact rule.
   if ! gcloud compute firewall-rules describe "${FIREWALL_SSH}" &>/dev/null; then
     echo "Creating new firewall for SSH: ${FIREWALL_SSH}" >&2
+    echo gcloud compute firewall-rules create "${FIREWALL_SSH}" \
```

Comment:
Yes. I figured it'd help debugging a bit, since you'd know exactly what the commands being run are. I can remove them if you feel they're unnecessary.